### PR TITLE
PAYEE-103: For working-hours, wrap the H1 inside the legend 

### PR DESCRIPTION
**Pull request by vivekhmrc**
It is so that the information is only conveyed to the user once

Signed-off-by: vivekhmrc <vivek.kumar@digital.hmrc.gov.uk>

- [x] Unit tests passing
- [ ] Coverage reached
- [x] Acceptance tests passing
- [x] Reviewed

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -1297,8 +1297,8 @@ tai.workingHours.legend=Your working hours are
 tai.workingHours.preHeading=Update your estimated income from {0}
 tai.workingHours.question=Are your working hours on each payslip generally the same or very different?
 tai.workingHours.regularHours=Hours are usually similar or the same from week to week
-tai.workingHours.heading=What are your working hours like through the year?
-tai.workingHours.title=What are your working hours like through the year?
+tai.workingHours.heading=What are your working hours through the year?
+tai.workingHours.title=What are your working hours through the year?
```
Comment (brendanstraughan) [inline conf/messages:1301]: We need the welsh translation for this

Comment (masahmad) [reply conf/messages:1301]: Thanks @brendanstraughan  - check commen t on ticket re: Welsh 
https://jira.tools.tax.service.gov.uk/browse/PAYEE-103

**Review by masahmad — APPROVED** [review]
LGTM

File: conf/messages (hunk 1 above)
Comment (brendanstraughan) [reply conf/messages:1301]: cool, will approve when I know a bit more about this force push

**Review by brendanstraughan — APPROVED** [review]

File: conf/messages (hunk 1 above)
Comment (vivekhmrc) [reply conf/messages:1301]: Discussed this with Suchet and the Welsh translation is correct

**Review by Jamie22Butler — APPROVED** [review]
LGTM :)

**Pull request merged**